### ISSUE TITLE
Support for USB re-insertion/removal of a webcam

### ISSUE THREAD

**Issue by emoto-yasushi** [homuler/MediaPipeUnityPlugin#1255]
### Feature Description

Currently, if tracking is started with the webcam connected and the USB connection of the webcam is removed, tracking will not resume even if the USB is reconnected.

As it stands, everything works fine as long as the camera icon button is pressed and the webcam is reselected. However, it would be great if tracking could automatically resume when the webcam is reconnected.

When the USB is disconnected, an error like "SampleCB() - buffer sizes do not match" appears, as shown in the following image.
For the testing, I am using Unitypackage v0.15.0, Unity 2022.3.6f1, and the sample scene of FaceLandmarkDetection.
![error](https://github.com/user-attachments/assets/d7ffda86-fa89-467b-bfe2-32f3a515356a)

**Comment by homuler** [comment]
I agree it would be a useful feature, but since this is just a sample application, I don't believe it's a high priority, and I'm not sure if I'll actually implement it (I prefer to eliminate non-essential code as much as possible).
If there are any specific reasons why it would be problematic not to have it, please let me know.
